content/docs/ethernet-v0.5/environment.md

```diff
@@ -0,0 +1,109 @@
+
+Everything except using the FPGA boards should also work
+out-of-box in a virtual machine.
+
+The following instructions are identical to the previous release, apart from the release name.
+For a more detailed explanation follow the
+[instructions of a previous tutorial](/docs/untether-v0.2/dev-env).
+
+Ensure you have all the necessary packages installed:
+
+    sudo apt-get install autoconf automake autotools-dev curl \
+    libmpc-dev libmpfr-dev libgmp-dev gawk build-essential bison \
+    flex texinfo gperf libncurses5-dev libusb-1.0-0 libboost-dev \
+    swig git libtool libreadline-dev libelf-dev python-dev \
+    microcom chrpath gawk texinfo nfs-kernel-server xinetd pseudo 
+
+Next steps:
+
+ * [Install FPGA and simulation tools](/docs/ethernet-v0.5/installtools)
+
+You might want to add the Vivado tools to your path first to keep the environment clean. This prevents system tools
+from trying to use shared libraries from the (older) Vivado install. Proceed as follows if you chose the default install
+location (or follow your system adminstrator instructions)
+
+    source /opt/Xilinx/Vivado/2015.4/settings64.sh
+    unset LD_LIBRARY_PATH
+
+### Download the code
+
+The code is hosted in the
+[lowRISC chip git repository](https://github.com/lowrisc/lowrisc-chip). All
+external repositories are fetched as submodules. In case you want to work on multiple branches
+give each checkout a unique name (such as lowrisc-chip-ethernet-v0.5)
+You need to clone the
+proper branch (`ethernet-v0.5`):
+
+    git clone -b ethernet-v0.5 --recursive https://github.com/lowrisc/lowrisc-chip.git lowrisc-chip-ethernet-v0.5
+    cd lowrisc-chip-ethernet-v0.5
+
+### Structure of the git repository
+
+The structure is similar to the one described
+[here](/docs/untether-v0.2/dev-env/#gitstruct). However
+the peripherals that were previously in the Minion repository are connected directly to the rocket.
+
+### Next steps
+    
+To set the correct environment variables for running lowRISC, you need to
+source the script `set_env.sh` (formerly `set_riscv_env.sh`) in the base directory.
+This must be done in a fresh shell window, not one used to work on any other branch of LowRISC:
+
+    source set_env.sh
+
+It is possible to override the default values by exporting variables before sourcing the script.
+The following variables are overridable:
+
+    $TOP                Path to the lowrisc-chip directory ($PWD).
+    $RISCV              Path to the riscv toolchain ($TOP/riscv).
+    $OSD_ROOT           Path to the Open SoC Debug tools ($TOP/tools).
+    $FPGA_BOARD         The target FPGA board (nexys4_ddr).
+
+## Download and build Linux and busybox (early stage user commands)
+
+This is not strictly needed unless we are building our own images, but it is easier to do it now.
+
+    sh $TOP/riscv-tools/fetch_and_patch_linux.sh
+
+## Building the RISCV toolchain (quick start: detailed instructions are below)
```

Comment:
Maybe explicitly say "This step is optional if you choose to use a pre-compiled rootfs."